src/Fantomas.Core/Context.fs

```diff
@@ -1295,3 +1295,28 @@ let colWithNlnWhenItemIsMultilineUsingConfig (items: ColMultilineItem list) (ctx
         colWithNlnWhenItemIsMultiline items ctx
     else
         col sepNln items (fun (ColMultilineItem (expr, _)) -> expr) ctx
+
```

Comment:
@jindraivanek thanks again for the tip, I came up with this solution.
Let me know if you see any room for improvement.